tests/playwright/src/model/workbench/status-bar.ts

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+
+import { closeConfirmationDialog, handleConfirmationDialog } from '../../utility/operations';
+import { BasePage } from '../pages/base-page';
+
+export class StatusBar extends BasePage {
```

Comment:
You need to add also this class to the `index.ts` so it will be part of npm `@podman-desktop/tests-playwright` package.